### PR TITLE
Fixes and enhancement

### LINKED ISSUE #1: Logic Improvement
There are many pssible ways you can think of.
You can try implementing char by char matching.
You can implement better methods for generating paragraphs.
Handle exceptional cases. The game does not work correctly if the paragraph has contiguous spaces.

### PR DESCRIPTION
**Pull request by iosdev747**
fix #1 and enhancement #2 , enhancement #3 , enhancement #4 and enhancement #5

### CODE REVIEW

File: Typeracer_Game/src/typeracer_game/GameStarter.java [hunk 1]
```diff
@@ -43,34 +54,118 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
...
+    private void quitButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_quitButtonActionPerformed
+        // TODO add your handling code here:
+        System.exit(0);
+    }//GEN-LAST:event_quitButtonActionPerformed
+
+    private void serverButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_serverButtonActionPerformed
+        // TODO add your handling code here:
+        messageLabel.setText("Server Started and Running ...");
+        messageLabel.paintImmediately(messageLabel.getVisibleRect());
+        GameServer object = new GameServer();
```
Comment (Rashi-G) [inline Typeracer_Game/src/typeracer_game/GameStarter.java:155]: GameServer class doesn't exist !??

Comment (iosdev747) [reply Typeracer_Game/src/typeracer_game/GameStarter.java:155]: sorry i forget to add them. Now i added these class

Comment (iosdev747) [reply Typeracer_Game/src/typeracer_game/GameStarter.java:155]: @Rashi-G 
i think the best final project is gonna to be merge at last?

Comment (iosdev747) [reply Typeracer_Game/src/typeracer_game/GameStarter.java:155]: whatever, in my code it also solved fix #6

Comment (Rashi-G) [reply Typeracer_Game/src/typeracer_game/GameStarter.java:155]: Nice! Let me have a look :)

**Comment by Rashi-G** [comment]
Ensure that when a game is played among multiple players, they all get same paragraph at same time. Then only you will be able to compare results.

**Comment by iosdev747** [comment]
it is something like a player joins a game and play it, while playing the current wpm is shared among all the online players.

**Comment by Rashi-G** [comment]
But there should also be a feature which allows player to compete among themselves. That's the essence of  'multiplayer'. Also 'leaderboard' and 'results' don't make sense if they don't get same paragraph at same time.

**Pull request merged**